app/Console/Commands/AddCoordinapeUserToAllCircles.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Profile;
+use App\Models\User;
+use App\Models\Circle;
+use Illuminate\Console\Command;
+use App\Repositories\CircleRepository;
+
+class AddCoordinapeUserToAllCircles extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'ape:populate_coordinape_user';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command to add a Coordinape user to all circles with address defined in ENV';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(CircleRepository $repo)
+    {
+        $this->repo = $repo;
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $address = config('ape.coordinape_user_address');
+
+        $profile = Profile::firstOrCreate([
+            'address' => $address,
+        ]);
+
+
+        # Iterate across all circles (TODO: can we do this in one transaction?)
```

Comment:
yeah. its possible to do it with DB::insert($array)

Reply:
or wrap in DB::transaction?

Reply:
lets do this later, i think its not needed for now.